.github/workflows/clippy_bors.yml

```diff
@@ -133,14 +133,6 @@ jobs:
       run: cargo test --features deny-warnings
       working-directory: clippy_dev
 
-    - name: Test cargo-clippy
-      run: ../target/debug/cargo-clippy
-      working-directory: clippy_workspace_tests
-
-    - name: Test cargo-clippy --fix
-      run: ../target/debug/cargo-clippy clippy --fix
-      working-directory: clippy_workspace_tests
-
```

Comment:
These could be replaced with the new directory. I'm just not sure what this is supposed to test, and the test should probably be contained in the new `workspace.rs`.

Reply:
Yeah, no idea why this is tested here.